pkg/sql/colexec/external/external.go

```diff
@@ -149,6 +150,56 @@ func InitS3Param(param *tree.ExternParam) error {
 	return nil
 }
 
+func judgeContainAccount(expr *plan.Expr) (bool, string) {
+	expr_F, ok := expr.Expr.(*plan.Expr_F)
+	if !ok {
+		return false, ""
+	}
+	if len(expr_F.F.Args) != 2 {
+		return false, ""
+	}
+	expr_Col, ok := expr_F.F.Args[0].Expr.(*plan.Expr_Col)
+	if !ok || !strings.Contains(expr_Col.Col.Name, "account") {
+		return false, ""
+	}
+	_, ok = expr_F.F.Args[1].Expr.(*plan.Expr_C)
+	if !ok {
+		return false, ""
+	}
+
+	str := expr_F.F.Args[1].Expr.(*plan.Expr_C).C.GetSval()
+	if str == "" {
+		return false, ""
+	}
+	return true, str
+}
+
+func FliterByAccount(node *plan.Node, proc *process.Process, fileList []string) ([]string, error) {
```

Comment:
This filter by account code should not be here.   This should be part of view definition by XZX.